README.md

```diff
@@ -1,264 +1,340 @@
+
+See memory requirements for the latest release [here](https://docs.aws.amazon.com/embedded-csdk/202011.00/lib-ref/libraries/aws/jobs-for-aws-iot-embedded-sdk/docs/doxygen/output/html/index.html#jobs_memory_requirements).
+
+
+### AWS IoT Device Defender Client
+
+
+[AWS IoT Device Defender Client](https://github.com/aws/device-defender-for-aws-iot-embedded-sdk) enables you to interact with AWS IoT Device Defender service to continuously monitor security metrics from devices for deviations from what you have defined as appropriate behavior for each device. If something doesn’t look right, AWS IoT Device Defender sends out an alert so you can take action to remediate the issue. More details about Device Defender can be found in [AWS IoT Device Defender documentation](https://docs.aws.amazon.com/iot/latest/developerguide/device-defender.html).
+
+### AWS Collection of Metrics
 
 AWS collects usage metrics indicating the operating system, hardware platform and MQTT client information of use to AWS IoT from the MQTT mutual auth demo by sending a specially formatted string in the username field of the MQTT CONNECT packet. These metrics help AWS IoT improve security and provide better technical support. Providing these metrics is optional for users, and can be disabled by updating the `OS_NAME`, `OS_VERSION`, `HARDWARE_PLATFORM_NAME` and `MQTT_LIB` configuration macros in the `demos/mqtt/mqtt_demo_mutual_auth/demo_config.h` file of the MQTT mutual auth demo.
 
 ### Format
 
 The format of the username string with metrics is:
+
 ```
 <Actual_Username>?SDK=<OS_Name>&Version=<OS_Version>&Platform=<Hardware_Platform>&MQTTLib=<MQTT_Library_name>@<MQTT_Library_version>
 ```
+
 where
 
-* **Actual_Username** is the actual username used for authentication (if a username/password is used for authentication).
-* **OS_Name** is the Operating System the application is running on.
-* **OS_Version** is the version number of the Operating System.
-* **Hardware_Platform** is the Hardware Platform the application is running on.
-* **MQTT_Library_name** is the MQTT Client library being used.
-* **MQTT_Library_version** is the version of the MQTT Client library being used.
+* *Actual_Username* is the actual username used for authentication (if a username/password is used for authentication).
+* *OS_Name* is the Operating System the application is running on.
+* *OS_Version* is the version number of the Operating System.
+* *Hardware_Platform* is the Hardware Platform the application is running on.
+* *MQTT_Library_name* is the MQTT Client library being used.
+* *MQTT_Library_version* is the version of the MQTT Client library being used.
```

Comment:
pls keep the same format as original. No italicsm, bold font as it shows in current readme file https://github.com/aws/aws-iot-device-sdk-embedded-C#format